### PR TITLE
#743 keyboard shortcuts help

### PR DESCRIPTION
**Pull request by gmarinov**

### CODE REVIEW

File: browser/scripts/ui.js [hunk 1]
```diff
@@ -621,6 +621,87 @@ VizorUI.prototype.updateProgressBar = function(percent) {
...
+			html.push(tag + key.toLowerCase() + end)
+		}
+
+		if (Object.prototype.toString.call(modifiedKeyCode) === '[object String]' ) { // isString()
+			key = modifiedKeyCode;
+			add(key.toLowerCase());
+		} else {
+			var key_meta = (isOSX) ? 'cmd' : 'ctrl';
+
+			if (modifiedKeyCode >= mod_alt) {
```
Comment (kschzt) [inline browser/scripts/ui.js]: mixing under_score and camelCase

File: browser/scripts/ui.js [hunk 2]
```diff
@@ -621,6 +621,87 @@ VizorUI.prototype.updateProgressBar = function(percent) {
...
+				add(key_meta, 'modifier key_ctrl key_cmd');
+				modifiedKeyCode -= mod_ctrl;
+			}
+			if (modifiedKeyCode >= mod_shift) {
+				add('shift', 'modifier key_shift');
+				modifiedKeyCode -= mod_shift;
+			}
+
+			var keyCodeMap = {	// keys without shift
+				186: 59,
```
Comment (kschzt) [inline browser/scripts/ui.js]: what keys are these?

**Comment by kschzt** [comment]
lgtm otherwise

File: browser/scripts/ui.js (hunk 2 above)
Comment (gmarinov) [inline browser/scripts/ui.js]: Good point. I'd like to stop processing keycodes in this primitive way altogether. Nothing I've found so far has been reliable.

**Pull request closed without merging**